lib/stream-posts.js

```diff
@@ -109,6 +109,7 @@ function pipeStream(
       buffer = initStream(config, brokerToken, streamingID, ioData, serverId);
     })
     .on('data', (chunk) => {
+      logResponse(logContext, '(streaming)', {body: chunk.toString()}, config);
```

Comment:
This will split large bodies across multiple log entries, and might cause problems if a binary file (e.g., a ZIP file) is logged (though that should be pretty rare for anything that's not Artifactory or Nexus). Also not thrilled about `status` being `(streaming)` - I'd be inclined to set it to `undefined` and just not include it, but otherwise approved.